examples/wCCD/src/wCCD.tsx

```diff
@@ -130,6 +131,19 @@ async function updateWCCDBalanceAccount(rpcClient: ConcordiumGRPCClient, account
     return BigInt(leb.decodeULEB128(toBuffer(ReturnValue.toHexString(res.returnValue).slice(4), 'hex'))[0]);
 }
 
+function addWCDToWallet(
+    _accountAddress: string,
+    tokenIds: string[],
+    contractAddressSource: number | bigint,
+    contractSubindex: number | bigint
+) {
+    const accountAddress = AccountAddress.fromBase58(_accountAddress);
+    const contractAddress = ContractAddress.create(contractAddressSource, contractSubindex);
+    detectConcordiumProvider().then((provider) => {
+        provider.addCIS2Tokens(accountAddress, tokenIds, contractAddress);
+    });
+}
```

Comment:
I would suggest using the client from the "wallet connector" instead of adding the browser wallet as a dependency.

`@concordium/react-components` is a react wrapper for `@concordium/wallet-connectors`, which aims to provide a common interface for interacting with concordium wallet connect wallets (mobile wallets) and the browser wallet.

You could do:
```suggestion
function addWCDToWallet(
    _accountAddress: string,
    client: WalletApi,
    tokenIds: string[],
    contractAddressSource: number | bigint,
    contractSubindex: number | bigint
) {
    const accountAddress = AccountAddress.fromBase58(_accountAddress);
    const contractAddress = ContractAddress.create(contractAddressSource, contractSubindex);
    return client.addCIS2Tokens(accountAddress, tokenIds, contractAddress);
}
```

Reply:
Thanks for the example. I didn't know that **client** can be received from **connection.client**